src/Faker/Provider/de_CH/Address.php

```diff
@@ -48,8 +48,8 @@ class Address extends \Faker\Provider\Address
      */
     protected static $canton = [
         ['AG' => 'Aargau'],
-        ['AI' => 'Appenzell Innerrhoden'],
-        ['AR' => 'Appenzell Ausserrhoden'],
+        ['AI' => 'Appenzell Innerrhoden'],
```

Comment:
What does this change do?

Reply:
Correction of a UTF non-standard whitespace character to a space. I would assume this error got introduced by someone editing the file with the editor using the wrong encoding.